### ISSUE TITLE
[Enhancement] Show info about track, composer, artist 

### ISSUE THREAD

**Issue by nicolas-raoul** [emijrp/wikiradio#7]
- Pictures of the composer and artist, of the album if available.
- Dates and locations at which the song was composed, recorded.
- Include Wikipedia articles text and images, if available.

This will probably be much easier to implement when structured Commons is available.

**Comment by wilfredor** [comment]
Yes, for the moment do something like that gone be another non maintainable parse

**Comment by prototyperspective** [comment]
This should be in the file title (if not move the file) and in the details that can be seen with the "More details >>" button.

Such picture are rarely available and like a link to the Wikipedia article could be included in the file description.
